javascript-local-storage/data.js

```diff
@@ -0,0 +1,12 @@
+/* exported todos */
+
+var todos = [];
+var previousTodosJSON = window.localStorage.getItem('javascript-local-storage');
+
+todos = JSON.parse(previousTodosJSON);
```

Comment:
What if there's nothing in localStorage? This would break the app permanently the first time the user uses the app.